windows/src/Defines.mak

```diff
@@ -28,50 +28,50 @@ INSTALLPATH_KEYMANDEVELOPER=%ProgramFiles(X86)%\Keyman\Keyman Developer
 INSTALLPATH_KEYMANENGINE=%CommonProgramFiles(X86)%\Keyman\Keyman Engine
 
 !IFDEF DEBUG
-  MAKEFLAG_DEBUG=-DDEBUG
-  DELPHI_MSBUILD_FLAG_DEBUG=/p:Config=Debug
+MAKEFLAG_DEBUG="DEBUG=$(DEBUG)"
+DELPHI_MSBUILD_FLAG_DEBUG="/p:Config=Debug"
 !ELSE
-  DELPHI_MSBUILD_FLAG_DEBUG=/p:Config=Release
+DELPHI_MSBUILD_FLAG_DEBUG="/p:Config=Release"
 !ENDIF
 
 !IFDEF USERDEFINES
-  MAKEFLAG_USERDEFINES=-DUSERDEFINES
+MAKEFLAG_USERDEFINES="USERDEFINES=$(USERDEFINES)"
```

Comment:
`-D` command line parameter to define a macro is replaced with equivalent macro definition